templates/index.html

```diff
@@ -1,211 +1,161 @@
-      <div class="u-sv3">
-        <p>Get help</p>
-        <div class="p-code-snippet">
-          <pre class="p-code-snippet__block--icon"><code>multipass help</code></pre>
-        </div>
-        <div class="p-code-snippet">
-          <pre class="p-code-snippet__block--icon"><code>multipass help &lt;command&gt;</code></pre>
-        </div>
-        <div class="u-sv3">
-          <p>Now don’t forget you still have <code>ubuntu-lts</code> running :)</p>
-        </div>
-      </div>
+<section class="p-strip">
+  <div class="row u-equal-height u-vertically-center">
+    <div class="col-5 u-align--center u-hide--small">
+      <img src="https://assets.ubuntu.com/v1/33d66ec7-Image-catalogue.svg" style="height: 171px; width: 300px;"/>
     </div>
+    <div class="col-7">
+      <h2>A curated catalogue of images</h2>
+      <p class="p-heading--4">Images with apps built-in one command away</p>
+      <p>Multipass is growing a catalogue of images to initialise pre-installed applications to get started with a single command.</p>
+      <a href="https://discourse.ubuntu.com/c/multipass/21" class="p-button--neutral p-link--external">Talk to us about your project</a>
+    </div>
+  </div>
 </section>
 
-<section class="p-strip--light" id="learn-more">
-  <div class="row">
-    <div class="col-12">
-      <h3 class="p-heading--two">Learn more</h3>
+<section class="p-strip--light">
+  <div class="row u-equal-height u-vertically-center">
+    <div class="col-7">
+      <h2>Appliances in VMs</h2>
+      <p class="p-heading--4">Ubuntu Appliance images to try before you Pi</p>
+      <p>Run a Virtual Ubuntu Appliance on your workstation without affecting your existing system. No need for spare hardware, just follow the steps and you’ll be up and running in minutes.</p>
+      <a href="https://ubuntu.com/appliance/vm" class="p-button--neutral p-link--external">Try Ubuntu Appliances</a>
+    </div>
+    <div class="col-5 u-align--center u-hide--small">
+      <img src="https://assets.ubuntu.com/v1/9c8d314f-what-is-an-appiance-header.svg" style="height: 246px; width: 300px;"/>
     </div>
   </div>
-  <div class="row p-divider">
-    <div class="col-6 p-divider__block">
-      <h4>Documentation</h4>
-      <p><a href="https://multipass.run/docs">Documentation and guidance on using Multipass&nbsp;›</a></p>
-    </div>
-    <div class="col-6 p-divider__block">
-      <h4>Participate</h4>
-      <ul class="p-list">
-        <li class="p-list__item">
-          <a href="https://webchat.freenode.net/?channels=#multipass" class="p-link--external">Multipass on Freenode</a>
-        </li>
-        <li class="p-list__item">
-          <a href="https://github.com/CanonicalLtd/multipass" class="p-link--external">Multipass on GitHub</a>
-        </li>
-        <li class="p-list__item">
-          <a href="https://discourse.ubuntu.com/c/multipass" class="p-link--external">Multipass Discourse</a>
-        </li>
-      </ul>
+</section>
+
+<section class="p-strip">
```

Comment:
I think you need to add `is-bordered` class here to separate from the footer

Reply:
Thanks, will do